libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -602,7 +597,7 @@ - (void)passcodeValidatedToAuthValidation
 
 - (void)authValidationAtLaunch
 {
-    if (![SFUserAccountManager sharedInstance].isCurrentUserAnonymous && ![SFUserAccountManager sharedInstance].currentUser.credentials.accessToken && self.authenticateAtLaunch) {
+    if (self.authenticateAtLaunch &&  ![SFUserAccountManager sharedInstance].currentUser.credentials.accessToken) {
```

Comment:
I thought we were trying to get rid of current user, or is this what's coming later and we will keep currentUser for a while